wagtail/wagtailcore/tests/test_blocks.py

```diff
@@ -26,6 +26,11 @@
 from wagtail.wagtailcore.rich_text import RichText
 
 
+# Skip the tests for Block.value_omitted_from_data on versions of Django that do not
+# implement Widget.value_omitted_from_data (scheduled to be added in Django 1.10.2)
+SKIP_VALUE_OMITTED_FROM_DATA_TESTS = not hasattr(forms.Widget, 'value_omitted_from_data')
```

Comment:
``django.VERSION >= (1, 10, 2)`` would make this check more greppable, which would be helpful when we remove Django 1.10 support.